core/src/main/java/hudson/PluginWrapper.java

```diff
@@ -87,6 +87,7 @@
  */
 @ExportedBean
 public class PluginWrapper implements Comparable<PluginWrapper>, ModelObject {
+    public static final boolean DISABLE_PLUGIN_DEPENDENCIES_VERSION_CHECK = Boolean.getBoolean(PluginWrapper.class.getName()+"." + "disablePluginDependenciesVersionCheck");
```

Comment:
Prefer to use positive, not negative, sense on options. Thus, a property which refers to _enabling_ the version check, defaulting to true.